components/camel-as2/camel-as2-api/src/main/java/org/apache/camel/component/as2/api/AS2ClientConnection.java

```diff
@@ -40,21 +50,46 @@
 import org.apache.http.protocol.RequestUserAgent;
 import org.apache.http.util.Args;
 
+import java.io.IOException;
```

Comment:
Please run the build with sourcecheck profile enabled and check the warnings. The imports order is wrong.

Reply:
Hi @oscerd, 

Where can I find the sourcecheck profile? I'm using Intellij

Reply:
Hello,

You just need to run the module build with -Psourcecheck. You'll see some warnings at the beginning.